docs/glossary.rst

```diff
@@ -94,6 +95,23 @@ email alert if something seems wrong. Only system admins connect
 to this server, and they may only do so using Tor.
 
 
+Onion Service
+-------------
+
+Tor onion services provide anonymous inbound connections to websites and other servers exclusively over the Tor network. For example, SecureDrop uses onion services for the *Journalist Interface* and *Source Interface* websites, as well as for adminsitrative access to the servers in SSH-over-Tor mode.
+
+Onion services can be accessed by clicking a link or pasting the onion service address into Tor Browser. For example, `sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion <https://sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion>`__ is SecureDrop's onion service address.
+
+Onion Service v2
+""""""""""""""""
+
+The third generation of onion services (v3) provides stronger cryptographic primitives (algorithms) than v2 onion services, and include redesigned protocols that guard against service information leaks on the Tor network.
```

Comment:
```suggestion
The third generation of onion services (v3) provides stronger cryptographic primitives (algorithms) than v2 onion services, and includes redesigned protocols that guard against service information leaks on the Tor network.
```

💡 @ninavizz This can be a convenient way to amend copy. In the "Files changed" tab of this PR, near the beginning of any line, you can make a little "+" button appear to leave a comment. In the comment composer, there is an icon representing a file with a +/- symbol that allows to **insert a suggestion**. Clicking it copies the line as-is and allows you to edit it at will. Once you post the comment, a diff is automatically generated to highlight your changes 🙂 

It only works on line that have already been changed (that appear in the "Files changed" tab), but the result can be committed directly using the GitHub interface, so it's quite handy for larger re-wordings.

Reply:
**Extra level**: if you drag the little "+" icon when leaving a comment, you can leave a comment on multiple lines at once. ✨

Reply:
(FYI, Nina is taking a break until later in January and may not be able respond here.)

IMO if we unpack "cryptographic primitives (algorithms)" we may as well just say "cryptographic algorithms".

Reply:
> Including words like "Cryptography Primitives" is discouraged, as so few users will understand what that means for the specificity to have value. I moved that bit to suggested text for the Onion Services glossary item. (@ninavizz's notes in #122)

I made the change to "cryptographic algorithms", though on second thought, I'm wondering if "**security features**" wouldn't be more accessible, even in the glossary.